core/data/src/main/java/com/example/poc/core/data/credentials/CredentialsRepository.kt

```diff
@@ -111,4 +112,15 @@ class CredentialsRepository(
     suspend fun deleteCredentials() = withContext(externalScope.coroutineContext) {
         credentialsLocalDataSource.deleteCredentials()
     }
+
+    suspend fun logout() {
```

Comment:
why do you need a separate method for logout? Wasn't deleteCredentials() created for that purpose?